pkg/apis/servicecatalog/types.go

```diff
@@ -204,6 +228,14 @@ const (
 	InstanceConditionDeprovisionFailed InstanceConditionType = "DeprovisioningFailed"
 )
 
+// BindingList is a list of Bindings
+type BindingList struct {
+	metav1.TypeMeta
+	metav1.ListMeta
+
+	Items []Binding
```

Comment:
Does this and similar fields above/below need struct tags for json encoding? IIRC no, but I want to make sure